cmd/agent/dist/conf.d/orchestrator_pod.d/conf.yaml.default

```diff
@@ -0,0 +1,4 @@
+init_config:
+instances:
+  -
```

Comment:
Could we keep this format consistent with the [other default files](https://github.com/DataDog/datadog-agent/pull/19478/files)?

Reply:
[66510cc](https://github.com/DataDog/datadog-agent/pull/19478/commits/66510cc8c48c5f1d234ff105baab88101a775713)